logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/TeamEventReceiver.kt

```diff
@@ -57,7 +58,19 @@ internal class TeamEventReceiverImpl(
             teamId = event.teamId,
             userId = event.memberId,
         )
-            .onFailure { kaliumLogger.e("$TAG - failure on member join event: $it") }
+            .onSuccess {
+                val logMap = mapOf(
+                    "event" to event.toLogMap(),
+                )
+                kaliumLogger.i("Success Handling Event: ${logMap.toJsonElement()}")
```

Comment:
Same on this class to extract the success handling to a function

Reply:
Done